Firestore/core/test/firebase/firestore/util/filesystem_test.cc

```diff
@@ -128,25 +143,40 @@ TEST(FilesystemTest, RecursivelyCreateDirFailure) {
   Status status = RecursivelyCreateDir(subdir);
   EXPECT_EQ(FirestoreErrorCode::FailedPrecondition, status.code());
 
-  status = RecursivelyDelete(dir);
-  EXPECT_OK(status);
+  EXPECT_OK(RecursivelyDelete(dir));
 }
 
 TEST(FilesystemTest, RecursivelyDelete) {
   Path tmp_dir = TempDir();
-  ASSERT_TRUE(Exists(tmp_dir));
+  ASSERT_OK(IsDirectory(tmp_dir));
 
   Path file = Path::JoinUtf8(tmp_dir, TestFilename());
-  Status status = RecursivelyDelete(file);
-  EXPECT_OK(status);
-  EXPECT_FALSE(Exists(file));
+  EXPECT_NOT_FOUND(IsDirectory(file));
+
+  // Deleting a something that doesn't exist should succeed.
```

Comment:
Ultranit: s/a something/something (unless I misunderstand the grammar)?

Reply:
Done.